tests/test_convertReferenceCatalog.py

```diff
@@ -95,6 +106,100 @@ def test_main_args_expanded_glob(self):
+                refSchema = _makeSchema(**argDict)
+                self.assertTrue("coord_ra" in refSchema)
+                self.assertTrue("coord_dec" in refSchema)
+                self.assertTrue("coord_raErr" in refSchema)
+                self.assertTrue("coord_decErr" in refSchema)
+                for filterName in filterNameList:
+                    fluxField = filterName + "_flux"
+                    self.assertIn(fluxField, refSchema)
+                    self.assertNotIn("x" + fluxField, refSchema)
+                    fluxErrField = fluxField + "Err"
+                    self.assertIn(fluxErrField, refSchema)
+                    self.assertEqual(getRefFluxField(refSchema, filterName), filterName + "_flux")
+                self.assertEqual("resolved" in refSchema, addIsResolved)
+                self.assertEqual("variable" in refSchema, addIsVariable)
+                self.assertEqual("photometric" in refSchema, addIsPhotometric)
+                self.assertEqual("photometric" in refSchema, addIsPhotometric)
+
+                # In the default schema none of the following should be
+                # included. We test setting these all together below.
+                self.assertNotIn("epoch", refSchema)
+                self.assertNotIn("pm_ra", refSchema)
+                self.assertNotIn("pm_dec", refSchema)
+                self.assertNotIn("pm_flag", refSchema)
+                self.assertNotIn("parallax", refSchema)
+                self.assertNotIn("parallax_flag", refSchema)
+
+    def testMakeSchema_fullCovariance(self):
+        """Make a schema with full position information and coordinate
+        covariance and test it."""
+        refSchema = _makeSchema(filterNameList=["r"], fullPositionInformation=True)
+        # Test that the epoch, proper motion and parallax terms are included in
+        # the schema.
+        self.assertIn("epoch", refSchema)
+        self.assertIn("pm_ra", refSchema)
+        self.assertIn("pm_dec", refSchema)
+        self.assertIn("pm_flag", refSchema)
+        self.assertIn("parallax", refSchema)
+        self.assertIn("parallax_flag", refSchema)
+        # Test that a sample of the 15 covariance terms are included in the schema.
+        self.assertIn("coord_raErr", refSchema)
+        self.assertIn("coord_decErr", refSchema)
+        self.assertIn("coord_ra_coord_dec_Cov", refSchema)
+        self.assertIn("pm_raErr", refSchema)
+        self.assertIn("pm_ra_parallax_Cov", refSchema)
+        self.assertIn("parallaxErr", refSchema)
+        self.assertEqual(refSchema['coord_raErr'].asField().getUnits(), "rad")
+        self.assertEqual(refSchema['coord_ra_coord_dec_Cov'].asField().getUnits(), "rad rad")
+        self.assertEqual(refSchema['pm_raErr'].asField().getUnits(), "rad/year")
+        self.assertEqual(refSchema['pm_dec_parallax_Cov'].asField().getUnits(), "rad/year rad")
+
+    def testCheckFluxUnits(self):
+        """Test that we can identify old style fluxes in a schema."""
```

Comment:
This needs to still live in the loader tests, it doesn't have anything to do with the converter.